source/_components/sensor.sigfox.markdown

```diff
@@ -0,0 +1,38 @@
+---
+layout: page
+title: "Sigfox Sensor"
+description: "Display messages from Sigfox devices in Home Assistant."
+date: 2018-04-07 12:10
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: file.png
```

Comment:
Incorrect logo. Maybe add the Sigfox logo to this PR?

Reply:
Could do, but does HA have a policy on seeking permission from the brand before using their logo?

Reply:
https://logos-download.com/11226-sigfox-logo-download.html

Reply:
OK I got permission from Sigfox to use one of their logos, will add tomorrow